frontend/test/testSetup.js

```diff
@@ -47,6 +49,17 @@ global.localVue.component('validation-observer', ValidationObserver)
 // global.localVue.directive('click-outside', clickOutside)
 global.localVue.directive('focus', focus)
 
+global.localVue.mixin(toasters)
+
+// Filter the warnings for portal vue
+const consoleWarn = global.console.warn
+// eslint-disable-next-line no-console
+delete console.warn
+// eslint-disable-next-line no-console
+console.warn = (m) => {
+  if (!m.match(/^\[portal-vue\]: Target .+ already exists$/)) consoleWarn(m)
```

Comment:
why is this happening?

Reply:
I do not know.
https://github.com/BeniRupp/bug_portal-vue-target-already-exists

Reply:
can you link this as comment?